ssg/rules.py

```diff
@@ -83,12 +83,15 @@ def get_rule_dir_ovals(dir_path, product=None):
     return results
 
 
-def find_rule_dirs(base_dir):
+def find_rule_dirs(base_dir, extra_dirs=None):
     """
-    Generator which yields all rule_directories within a given base_dir
+    Generator which yields all rule_directories within a given base_dir and, optionally, extra_dirs directories list
     """
-    for root, dirs, _ in os.walk(base_dir):
```

Comment:
ditto, please rename the original function and call it from the new `find_rule_dirs`.